### PR TITLE
fix: 1. virtual table will remove the row being dragged

### PR DESCRIPTION
**Pull request by CapryChen**

### CODE REVIEW

File: src/components/DragDropProvider.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ const DragDropProvider: FC<{
   const edgeDraggableIdRef = useRef<string>();
 
   const draggingIdRef = useRef<string>();
+  const draggingIdNodeRef = useRef<HTMLElement | null | undefined>();
```
Comment (Asweallcan) [inline src/components/DragDropProvider.tsx:65]: draggingNodeRef?

File: src/components/DragDropProvider.tsx [hunk 2]
```diff
@@ -62,6 +62,7 @@ const DragDropProvider: FC<{
   const edgeDraggableIdRef = useRef<string>();
 
   const draggingIdRef = useRef<string>();
```
Comment (Asweallcan) [inline src/components/DragDropProvider.tsx:64]: draggingIdRef seems useless, maybe remove it?

**Comment by SunPeter** [comment]
先 merge 到 aPaaS 分支，发一个 alpha 版本

File: src/components/DragDropProvider.tsx (hunk 2 above)
Comment (CapryChen) [reply src/components/DragDropProvider.tsx:64]: yes, I will updated recently。

**Comment by SunPeter** [comment]
人工 CI 
image    
https://www.npmjs.com/package/react-flex-dnd?activeTab=versions  
合 master 发正式版本还是你操作吧 @Asweallcan

**Comment by Asweallcan** [comment]
OK

**Pull request merged**